### ISSUE TITLE
Broken on any version of IE since v7.0.0

### ISSUE THREAD

**Issue by tiagomapmarques** [taylorhakes/promise-polyfill#67]
I'm requiring your project (`v7.0.0`) on mine along with `whatwg-fetch` `v2.0.3` and I'm applying the polyfill myself with:
```javascript
if (!window.Promise) {
  window.Promise = promisePolyfill;
  window.Promise._immediateFn = setImmediate; // this is the "setasap" package
}
require("whatwg-fetch"); // applies polyfill automatically
```

When I use my application on any IE browser, any `fetch` functions created will throw this error:
```
TypeError: Object doesn't support this action
```

Have confirmed that the exact same code does work with `v6.0.2`.
(version of `whatwg-fetch` did not change).

**Comment by dustinaevans** [comment]
Same here

**Comment by dustinaevans** [comment]
Resolved with 7.0.1

**Comment by tiagomapmarques** [comment]
Confirmed it is resolved. Thanks for the quick fix @taylorhakes :)
Closing the issue.
